src/foss-update.php

```diff
@@ -46,6 +46,18 @@ public function patch()
     }
 }
 
+/**
+ * Migration steps to create table to allow admin users to do password reset
+ */
+class FOSSPatch_26 extends FOSSPatchAbstract
```

Comment:
There is Allready an older class with the same name. This should become FOSSPatch_27

Reply:
will fix.